# Request
float64 px
float64 py
float64 pz

---
# Feedback
float64 percentage_complete

---
# Result
bool success
